src/actions/clusterActions.js

```diff
@@ -393,7 +403,7 @@ export function clusterCreate(cluster) {
       cluster,
     });
 
-    var clustersApi = new GiantSwarmV4.ClustersApi();
+    var clustersApi = new GiantSwarm.ClustersApi();
```

Comment:
I think it would be good to separate this name change into its own PR.

Reply:
@marian is this something I can PR against master?

Reply:
We'd have to get the same change in giantswarm-js-client as well, so if that can be done there too, seperate of the nodepools story, then that would be best.